ra2ce/analysis/direct/__init__.py

```diff
@@ -0,0 +1,2 @@
+from ra2ce.analysis.direct.direct_damage import DirectDamage
+from ra2ce.analysis.direct.effectiveness_measures import EffectivenessMeasures
```

Comment:
Though useful, these sort of imports can bring circular dependencies, be aware of it. Normally we will do explicit import declaration.

Reply:
I did this to facilitate the imports in e.g. the factory. I saw this in other `__init__.py` files. What might be a valid reason to do this?

Reply:
Usually you do this when the package to be imported is isolated, so there are no dependencies between the modules within it. Though I don't say to remove this, I suggest to stop "using" this approach as in other projects it ended up being counter-productive.

Reply:
Adjusted